frontend/src/metabase/lib/revisions/revisions.js

```diff
@@ -97,8 +97,8 @@ const CHANGE_DESCRIPTIONS = {
   dataset: {
     [CHANGE_TYPE.UPDATE]: (wasDataset, isDataset) =>
       isDataset
-        ? t`turned this into a dataset`
-        : t`reverted this from a dataset to a saved question`,
+        ? t`turned this into a model`
+        : t`reverted this from a model to a saved question`,
```

Comment:
I'm now trying to avoid the word "revert" for this action to prevent confusion about it being a "true" revert, since it isn't one as you pointed out.

```suggestion
        : t`changed this from a model to a saved question`,
```

Reply:
The fact remains that there is still a "revert" button.
But if this helps, I'm happy to commit these changes.